internal/shared/services/service/prices.go

```diff
@@ -10,10 +10,16 @@ import (
 	"go.mongodb.org/mongo-driver/mongo"
 )
 
+const (
+	// CoinMarketCap UCID for BABY.
+	// https://coinmarketcap.com/currencies/babylon/
+	BABY_COINMARKETCAP_ID = 32198
```

Comment:
in go constants usually written in CamelCase (`BabyCoinmarketcapID`), can we please change it here in other places ?